### PR TITLE
WiXv4 upgrade test

### PR DESCRIPTION
**Pull request by elaboy**
I upgraded the WiXV3 to WiXV4 using the wix convert command

### CODE REVIEW

**Review by Alexander-Sol — CHANGES_REQUESTED** [review]
The .msi artifact created by Appveyor from this PR fails to install a working version of MetaMorpheus. The installer runs, but any attempts to launch the newly installed version fails without giving an error message. The program simply doesn't open. 

From my experience working with WIX, this isn't uncommon. However, I don't know how to trouble shoot it

**Comment by trishorts** [comment]
one thing we may need the guid to pass microscoft thing. i did successfully install this previously though i haven't tried this incarnation.

File: MetaMorpheusSetup/MetaMorpheusSetup.wixproj [hunk 1]
```diff
@@ -1,31 +1,17 @@
-<?xml version="1.0" encoding="utf-8"?>
-<Project ToolsVersion="4.0" DefaultTargets="Build" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
+<Project>
+  <Import Project="Sdk.props" Sdk="WixToolset.Sdk" Version="4.0.0-rc.1" />
   <PropertyGroup>
-    <Configuration Condition=" '$(Configuration)' == '' ">Debug</Configuration>
     <Platform Condition=" '$(Platform)' == '' ">x64</Platform>
-    <ProductVersion>3.10</ProductVersion>
-    <ProjectGuid>be72541a-66a4-4958-9d19-56acc7d3876b</ProjectGuid>
```
Comment (trishorts) [inline MetaMorpheusSetup/MetaMorpheusSetup.wixproj:7]: may need this

File: Bootstrapper1/Bootstrapper1.wixproj [hunk 2]
```diff
@@ -1,32 +1,14 @@
-<?xml version="1.0" encoding="utf-8"?>
-<Project ToolsVersion="4.0" DefaultTargets="Build" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
+<Project Sdk="WixToolset.Sdk/4.0.0-rc.1">
   <PropertyGroup>
-    <Configuration Condition=" '$(Configuration)' == '' ">Debug</Configuration>
-    <Platform Condition=" '$(Platform)' == '' ">x86</Platform>
-    <ProductVersion>3.10</ProductVersion>
-    <ProjectGuid>{fba0315b-076d-4470-afb0-64c8ab4a65a9}</ProjectGuid>
```
Comment (trishorts) [inline Bootstrapper1/Bootstrapper1.wixproj:7]: maybe need this

**Pull request closed without merging**